x-pack/plugins/ml/public/application/jobs/new_job/common/chart_loader/chart_loader.ts

```diff
@@ -31,16 +31,16 @@ export type LineChartData = Record<DetectorIndex, LineChartPoint[]>;
 
 const eq = (newArgs: any[], lastArgs: any[]) => isEqual(newArgs, lastArgs);
 
-const newJobLineChart = memoizeOne(ml.jobs.newJobLineChart, eq);
-const newJobPopulationsChart = memoizeOne(ml.jobs.newJobPopulationsChart, eq);
-const getEventRateData = memoizeOne(mlResultsService.getEventRateData, eq);
-const getCategoryFields = memoizeOne(getCategoryFieldsOrig, eq);
-
 export class ChartLoader {
   private _indexPatternTitle: IndexPatternTitle = '';
   private _timeFieldName: string = '';
   private _query: object = {};
 
+  private newJobLineChart = memoizeOne(ml.jobs.newJobLineChart, eq);
```

Comment:
as these have been moved inside the class as private variables they should be renamed `_newJobLineChart` etc

Reply:
Renamed in https://github.com/elastic/kibana/pull/121652/commits/04c89c262f50a689b59fee30ecdca7e266d99cbe (but I actually don't think this naming convention should be applied to private fields 🙂 )